sparkles/tests/test_review.py

```diff
@@ -172,12 +172,12 @@ def test_catch_exception_from_method():
         acar.run_aca_review(roll_level='BAD VALUE')
 
 
-def test_run_aca_review_function():
+def test_run_aca_review_function(tmpdir):
```

Comment:
Looking back on this PR, I'm not really sure what this test is supposed to test.  If the acars are supplied to run_aca_review below, do we need to make_html?  Seems like the tmpdir could be cut and make_html=False supplied for this particular test.

Reply:
See https://github.com/sot/sparkles/pull/120#issuecomment-499720411, but obviously something is wrong.

Reply:
OK.  I'll figure out how to get the tmpdir to work for the outputs.

Reply:
Fix on the way.